finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -130,7 +130,7 @@ object HighResTimer {
    */
   val Default: com.twitter.util.Timer =
     new JavaTimer(true, Some("HighResTimer")) {
```

Comment:
if we're making it stoppable, we should also remove the { } from the construction

Reply:
Let's think about it one more time.
How can I stop this timer accidentally using dedicated  method `stop`.?
In my opinion people expect such behavior. If I call stop this timer should be stopped.
I thing that previous behavior  was quite unexpected.